### PR TITLE
Feature/intygfv 15093 ovrig kommentar

### PR DESCRIPTION
**Pull request by IamFriberg**

### CODE REVIEW

File: ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/codes/RespConstantsV7.java [hunk 1]
```diff
@@ -204,6 +204,85 @@ public final class RespConstantsV7 {
     public static final String TECKEN_SVIKTANDE_KOGNITIV_FUNKTION_JSON_ID = "sviktandeKognitivFormaga";
     public static final String TECKEN_SVIKTANDE_KOGNITIV_FUNKTION_SVAR_TEXT_ID = "FRG_23.RBK";
 
+    public static final String TECKEN_SOMN_ELLER_VAKENHETSSTORNING_CATEGORY_ID = "somnellervakenhetsstorning";
+    public static final String TECKEN_SOMN_ELLER_VAKENHETSSTORNING_CATEGORY_TEXT_ID = "KAT_10.RBK";
+    public static final String TECKEN_SOMN_ELLER_VAKENHETSSTORNING_SVAR_ID_24 = "24";
```
Comment (ChrilleSigerhed) [inline ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/codes/RespConstantsV7.java:209]: Should we remove the trailing number from the constant to keep the format same for all?

File: ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionAdhdAddDampAsbergersTourettes.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+import se.inera.intyg.common.support.facade.model.config.CertificateDataConfigRadioBoolean;
+import se.inera.intyg.common.support.facade.model.validation.CertificateDataValidation;
+import se.inera.intyg.common.support.facade.model.validation.CertificateDataValidationMandatory;
+import se.inera.intyg.common.support.facade.model.value.CertificateDataValueBoolean;
+import se.inera.intyg.common.ts_bas.v7.model.internal.Utvecklingsstorning;
+
+public class QuestionAdhdAddDampAsbergersTourettes {
+
+    public static CertificateDataElement toCertificate(Utvecklingsstorning utvecklingsstorning, int index, CertificateTextProvider textProvider) {
+        var harSyndrom = utvecklingsstorning != null ? utvecklingsstorning.getHarSyndrom() : null;
```
Comment (ChrilleSigerhed) [inline ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionAdhdAddDampAsbergersTourettes.java:43]: should this be final var?

File: ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionAdhdAddDampAsbergersTourettes.java [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+                CertificateDataValueBoolean.builder()
+                    .id(ADHD_ADD_DAMP_ASPERGERS_TOURETTES_DELSVAR_JSON_ID_29)
+                    .selected(harSyndrom)
+                    .build()
+            )
+            .validation(
+                new CertificateDataValidation[]{
+                    CertificateDataValidationMandatory.builder()
+                        .questionId(ADHD_ADD_DAMP_ASPERGERS_TOURETTES_DELSVAR_ID_29)
+                        .expression(singleExpression(ADHD_ADD_DAMP_ASPERGERS_TOURETTES_DELSVAR_JSON_ID_29))
```
Comment (ChrilleSigerhed) [inline ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionAdhdAddDampAsbergersTourettes.java:67]: Remove whitespace

File: ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionAlkoholNarkotikaJournaluppgifter.java [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+import se.inera.intyg.common.support.facade.model.config.CertificateDataConfigRadioBoolean;
+import se.inera.intyg.common.support.facade.model.validation.CertificateDataValidation;
+import se.inera.intyg.common.support.facade.model.validation.CertificateDataValidationMandatory;
+import se.inera.intyg.common.support.facade.model.value.CertificateDataValueBoolean;
+import se.inera.intyg.common.ts_bas.v7.model.internal.NarkotikaLakemedel;
+
+public class QuestionAlkoholNarkotikaJournaluppgifter {
+
+    public static CertificateDataElement toCertificate(NarkotikaLakemedel narkotikaLakemedel, int index, CertificateTextProvider textProvider) {
+        var getTeckenMissbruk = narkotikaLakemedel != null ? narkotikaLakemedel.getTeckenMissbruk() : null;
```
Comment (ChrilleSigerhed) [inline ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionAlkoholNarkotikaJournaluppgifter.java:43]: should this be final var?

File: ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionAlkoholNarkotikaLakarordinerat.java [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+import se.inera.intyg.common.support.facade.model.config.CertificateDataConfigRadioBoolean;
+import se.inera.intyg.common.support.facade.model.validation.CertificateDataValidation;
+import se.inera.intyg.common.support.facade.model.validation.CertificateDataValidationMandatory;
+import se.inera.intyg.common.support.facade.model.value.CertificateDataValueBoolean;
+import se.inera.intyg.common.ts_bas.v7.model.internal.NarkotikaLakemedel;
+
+public class QuestionAlkoholNarkotikaLakarordinerat {
+
+    public static CertificateDataElement toCertificate(NarkotikaLakemedel narkotikaLakemedel, int index, CertificateTextProvider textProvider) {
+        var lakarordineratLakemedelsbruk = narkotikaLakemedel != null ? narkotikaLakemedel.getLakarordineratLakemedelsbruk() : null;
```
Comment (ChrilleSigerhed) [inline ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionAlkoholNarkotikaLakarordinerat.java:43]: should this be final var?

File: ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionAlkoholNarkotikaOrdineratLakamedel.java [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+import se.inera.intyg.common.support.facade.model.validation.CertificateDataValidationText;
+import se.inera.intyg.common.support.facade.model.value.CertificateDataTextValue;
+import se.inera.intyg.common.ts_bas.v7.model.internal.NarkotikaLakemedel;
+
+public class QuestionAlkoholNarkotikaOrdineratLakamedel {
+
+    private static final short TEXT_LIMIT = 180;
+
+    public static CertificateDataElement toCertificate(NarkotikaLakemedel narkotikaLakemedel, int index, CertificateTextProvider textProvider) {
+        var lakemedelOchDos = narkotikaLakemedel != null ? narkotikaLakemedel.getLakemedelOchDos() : null;
```
Comment (ChrilleSigerhed) [inline ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionAlkoholNarkotikaOrdineratLakamedel.java:47]: should this be final var?

File: ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionAlkoholNarkotikaProvtagning.java [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+import se.inera.intyg.common.support.facade.model.config.CertificateDataConfigRadioBoolean;
+import se.inera.intyg.common.support.facade.model.validation.CertificateDataValidation;
+import se.inera.intyg.common.support.facade.model.validation.CertificateDataValidationMandatory;
+import se.inera.intyg.common.support.facade.model.value.CertificateDataValueBoolean;
+import se.inera.intyg.common.ts_bas.v7.model.internal.NarkotikaLakemedel;
+
+public class QuestionAlkoholNarkotikaProvtagning {
+
+    public static CertificateDataElement toCertificate(NarkotikaLakemedel narkotikaLakemedel, int index, CertificateTextProvider textProvider) {
+        var behovsProvtagning = narkotikaLakemedel != null ? narkotikaLakemedel.getProvtagningBehovs() : null;
```
Comment (ChrilleSigerhed) [inline ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionAlkoholNarkotikaProvtagning.java:43]: should this be final var?

File: ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionAlkoholNarkotikaVardinsatser.java [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+import se.inera.intyg.common.support.facade.model.config.CertificateDataConfigRadioBoolean;
+import se.inera.intyg.common.support.facade.model.validation.CertificateDataValidation;
+import se.inera.intyg.common.support.facade.model.validation.CertificateDataValidationMandatory;
+import se.inera.intyg.common.support.facade.model.value.CertificateDataValueBoolean;
+import se.inera.intyg.common.ts_bas.v7.model.internal.NarkotikaLakemedel;
+
+public class QuestionAlkoholNarkotikaVardinsatser {
+
+    public static CertificateDataElement toCertificate(NarkotikaLakemedel narkotikaLakemedel, int index, CertificateTextProvider textProvider) {
+        var faremalForVardinsatser = narkotikaLakemedel != null ? narkotikaLakemedel.getForemalForVardinsats() : null;
```
Comment (ChrilleSigerhed) [inline ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionAlkoholNarkotikaVardinsatser.java:43]: should this be final var?

File: ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionPsykiskUtvecklingsstorning.java [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+import se.inera.intyg.common.support.facade.model.config.CertificateDataConfigRadioBoolean;
+import se.inera.intyg.common.support.facade.model.validation.CertificateDataValidation;
+import se.inera.intyg.common.support.facade.model.validation.CertificateDataValidationMandatory;
+import se.inera.intyg.common.support.facade.model.value.CertificateDataValueBoolean;
+import se.inera.intyg.common.ts_bas.v7.model.internal.Utvecklingsstorning;
+
+public class QuestionPsykiskUtvecklingsstorning {
+
+    public static CertificateDataElement toCertificate(Utvecklingsstorning utvecklingsstorning, int index, CertificateTextProvider textProvider) {
+        var psykiskUtvecklingsstorning = utvecklingsstorning != null ? utvecklingsstorning.getPsykiskUtvecklingsstorning() : null;
```
Comment (ChrilleSigerhed) [inline ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionPsykiskUtvecklingsstorning.java:43]: should this be final var?

File: ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionPsykiskSjukdomStorning.java [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+import se.inera.intyg.common.support.facade.model.config.CertificateDataConfigRadioBoolean;
+import se.inera.intyg.common.support.facade.model.validation.CertificateDataValidation;
+import se.inera.intyg.common.support.facade.model.validation.CertificateDataValidationMandatory;
+import se.inera.intyg.common.support.facade.model.value.CertificateDataValueBoolean;
+import se.inera.intyg.common.ts_bas.v7.model.internal.Psykiskt;
+
+public class QuestionPsykiskSjukdomStorning {
+
+    public static CertificateDataElement toCertificate(Psykiskt psykiskt, int index, CertificateTextProvider textProvider) {
+        var psykiskSjukdom = psykiskt != null ? psykiskt.getPsykiskSjukdom() : null;
```
Comment (ChrilleSigerhed) [inline ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionPsykiskSjukdomStorning.java:43]: should this be final var?

File: ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionSomnOchVakenhetsstorningar.java [hunk 11]
```diff
@@ -0,0 +1,77 @@
...
+import se.inera.intyg.common.support.facade.model.config.CertificateDataConfigRadioBoolean;
+import se.inera.intyg.common.support.facade.model.validation.CertificateDataValidation;
+import se.inera.intyg.common.support.facade.model.validation.CertificateDataValidationMandatory;
+import se.inera.intyg.common.support.facade.model.value.CertificateDataValueBoolean;
+import se.inera.intyg.common.ts_bas.v7.model.internal.SomnVakenhet;
+
+public class QuestionSomnOchVakenhetsstorningar {
+
+    public static CertificateDataElement toCertificate(SomnVakenhet somnVakenhet, int index, CertificateTextProvider textProvider) {
+        var teckenSomnstorningar = somnVakenhet != null ? somnVakenhet.getTeckenSomnstorningar() : null;
```
Comment (ChrilleSigerhed) [inline ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionSomnOchVakenhetsstorningar.java:43]: should this be final var?

File: ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionStadigvarandeMedicinering.java [hunk 12]
```diff
@@ -0,0 +1,77 @@
...
+import se.inera.intyg.common.support.facade.model.config.CertificateDataConfigRadioBoolean;
+import se.inera.intyg.common.support.facade.model.validation.CertificateDataValidation;
+import se.inera.intyg.common.support.facade.model.validation.CertificateDataValidationMandatory;
+import se.inera.intyg.common.support.facade.model.value.CertificateDataValueBoolean;
+import se.inera.intyg.common.ts_bas.v7.model.internal.Medicinering;
+
+public class QuestionStadigvarandeMedicinering {
+
+    public static CertificateDataElement toCertificate(Medicinering medicinering, int index, CertificateTextProvider textProvider) {
+        var stadigvarandeMedicinering = medicinering != null ? medicinering.getStadigvarandeMedicinering() : null;
```
Comment (ChrilleSigerhed) [inline ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionStadigvarandeMedicinering.java:43]: should this be final var?

File: ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionStadigvarandeMedicineringBeskrivning.java [hunk 13]
```diff
@@ -0,0 +1,87 @@
...
+import se.inera.intyg.common.support.facade.model.validation.CertificateDataValidationText;
+import se.inera.intyg.common.support.facade.model.value.CertificateDataTextValue;
+import se.inera.intyg.common.ts_bas.v7.model.internal.Medicinering;
+
+public class QuestionStadigvarandeMedicineringBeskrivning {
+
+    private static final short TEXT_LIMIT = 180;
+
+    public static CertificateDataElement toCertificate(Medicinering medicinering, int index, CertificateTextProvider textProvider) {
+        var beskrivning = medicinering != null ? medicinering.getBeskrivning() : null;
```
Comment (ChrilleSigerhed) [inline ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionStadigvarandeMedicineringBeskrivning.java:47]: should this be final var?

File: ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionTidpunktVardPaSjukhus.java [hunk 14]
```diff
@@ -0,0 +1,88 @@
...
+import se.inera.intyg.common.support.facade.model.validation.CertificateDataValidationShow;
+import se.inera.intyg.common.support.facade.model.validation.CertificateDataValidationText;
+import se.inera.intyg.common.support.facade.model.value.CertificateDataTextValue;
+import se.inera.intyg.common.ts_bas.v7.model.internal.Sjukhusvard;
+
+public class QuestionTidpunktVardPaSjukhus {
+    private static final short TEXT_LIMIT = 40;
+
+    public static CertificateDataElement toCertificate(Sjukhusvard sjukhusvard, int index, CertificateTextProvider textProvider) {
+        var tidpunkt = sjukhusvard != null ? sjukhusvard.getTidpunkt() : null;
```
Comment (ChrilleSigerhed) [inline ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionTidpunktVardPaSjukhus.java:48]: should this be final var?

File: ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionVardatsPaSjukhus.java [hunk 15]
```diff
@@ -0,0 +1,76 @@
...
+import se.inera.intyg.common.support.facade.model.CertificateDataElement;
+import se.inera.intyg.common.support.facade.model.config.CertificateDataConfigRadioBoolean;
+import se.inera.intyg.common.support.facade.model.validation.CertificateDataValidation;
+import se.inera.intyg.common.support.facade.model.validation.CertificateDataValidationMandatory;
+import se.inera.intyg.common.support.facade.model.value.CertificateDataValueBoolean;
+import se.inera.intyg.common.ts_bas.v7.model.internal.Sjukhusvard;
+
+public class QuestionVardatsPaSjukhus {
+    public static CertificateDataElement toCertificate(Sjukhusvard sjukhusvard, int index, CertificateTextProvider textProvider) {
+        var sjukhusEllerLakarkontakt = sjukhusvard != null ? sjukhusvard.getSjukhusEllerLakarkontakt() : null;
```
Comment (ChrilleSigerhed) [inline ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionVardatsPaSjukhus.java:42]: should this be final var?

File: ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionVardatsPaSjukhusOrsak.java [hunk 16]
```diff
@@ -0,0 +1,87 @@
...
+import se.inera.intyg.common.support.facade.model.validation.CertificateDataValidationText;
+import se.inera.intyg.common.support.facade.model.value.CertificateDataTextValue;
+import se.inera.intyg.common.ts_bas.v7.model.internal.Sjukhusvard;
+
+public class QuestionVardatsPaSjukhusOrsak {
+
+    private static final short TEXT_LIMIT = 50;
+
+    public static CertificateDataElement toCertificate(Sjukhusvard sjukhusvard, int index, CertificateTextProvider textProvider) {
+        var anledning = sjukhusvard != null ? sjukhusvard.getAnledning() : null;
```
Comment (ChrilleSigerhed) [inline ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionVardatsPaSjukhusOrsak.java:47]: should this be final var?

File: ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionVardinrattningensNamn.java [hunk 17]
```diff
@@ -0,0 +1,87 @@
...
+import se.inera.intyg.common.support.facade.model.validation.CertificateDataValidationText;
+import se.inera.intyg.common.support.facade.model.value.CertificateDataTextValue;
+import se.inera.intyg.common.ts_bas.v7.model.internal.Sjukhusvard;
+
+public class QuestionVardinrattningensNamn {
+
+    private static final short TEXT_LIMIT = 40;
+
+    public static CertificateDataElement toCertificate(Sjukhusvard sjukhusvard, int index, CertificateTextProvider textProvider) {
+        var tidpunkt = sjukhusvard != null ? sjukhusvard.getVardinrattning() : null;
```
Comment (ChrilleSigerhed) [inline ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionVardinrattningensNamn.java:47]: should this be final var?

**Review by ChrilleSigerhed — CHANGES_REQUESTED** [review]
Wow, impressive last(?) PR. I think this looks great, my comments are mostly related to formatting. Please take a look and consider makeing the requested changes! :-)

Also, please decorate validationErrors in InternalValidatorInstance with questionId!

File: ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/codes/RespConstantsV7.java (hunk 1 above)
Comment (IamFriberg) [reply ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/codes/RespConstantsV7.java:209]: Will fix!

File: ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionAdhdAddDampAsbergersTourettes.java (hunk 3 above)
Comment (IamFriberg) [reply ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionAdhdAddDampAsbergersTourettes.java]: Fixed!

File: ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionAdhdAddDampAsbergersTourettes.java (hunk 2 above)
Comment (IamFriberg) [reply ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionAdhdAddDampAsbergersTourettes.java:44]: Fixed!

File: ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionAlkoholNarkotikaJournaluppgifter.java (hunk 4 above)
Comment (IamFriberg) [reply ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionAlkoholNarkotikaJournaluppgifter.java:44]: Fixed!

File: ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionAlkoholNarkotikaLakarordinerat.java (hunk 5 above)
Comment (IamFriberg) [reply ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionAlkoholNarkotikaLakarordinerat.java:44]: Fixed!

File: ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionAlkoholNarkotikaOrdineratLakamedel.java (hunk 6 above)
Comment (IamFriberg) [reply ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionAlkoholNarkotikaOrdineratLakamedel.java:48]: Fixed!

File: ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionAlkoholNarkotikaProvtagning.java (hunk 7 above)
Comment (IamFriberg) [reply ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionAlkoholNarkotikaProvtagning.java:44]: Fixed!

File: ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionAlkoholNarkotikaVardinsatser.java (hunk 8 above)
Comment (IamFriberg) [reply ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionAlkoholNarkotikaVardinsatser.java:44]: Fixed!

File: ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionPsykiskUtvecklingsstorning.java (hunk 9 above)
Comment (IamFriberg) [reply ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionPsykiskUtvecklingsstorning.java:44]: Fixed!

File: ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionPsykiskSjukdomStorning.java (hunk 10 above)
Comment (IamFriberg) [reply ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionPsykiskSjukdomStorning.java:43]: Fixed!

File: ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionSomnOchVakenhetsstorningar.java (hunk 11 above)
Comment (IamFriberg) [reply ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionSomnOchVakenhetsstorningar.java:43]: Fixed!

File: ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionStadigvarandeMedicinering.java (hunk 12 above)
Comment (IamFriberg) [reply ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionStadigvarandeMedicinering.java:43]: Fixed!

File: ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionStadigvarandeMedicineringBeskrivning.java (hunk 13 above)
Comment (IamFriberg) [reply ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionStadigvarandeMedicineringBeskrivning.java:47]: Fixed!

File: ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionTidpunktVardPaSjukhus.java (hunk 14 above)
Comment (IamFriberg) [reply ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionTidpunktVardPaSjukhus.java:46]: Fixed!

File: ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionVardatsPaSjukhus.java (hunk 15 above)
Comment (IamFriberg) [reply ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionVardatsPaSjukhus.java:42]: Fixed!

File: ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionVardatsPaSjukhusOrsak.java (hunk 16 above)
Comment (IamFriberg) [reply ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionVardatsPaSjukhusOrsak.java:47]: Fixed!

File: ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionVardinrattningensNamn.java (hunk 17 above)
Comment (IamFriberg) [reply ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionVardinrattningensNamn.java:47]: Fixed!

**Review by ChrilleSigerhed — APPROVED** [review]
Great work. After manually testing throught this PR we can now consider this done.

**Pull request merged**